packages/react/src/React.js

```diff
@@ -145,4 +148,8 @@ if (enableJSXTransformAPI) {
   }
 }
 
+if (enableAvoidThisRenderAPI) {
+  React.unstable_avoidThisRender = unstable_avoidThisRender;
```

Comment:
This `React.unstable_avoidThisRender` is the only place you need the `unstable_` prefix.